docs/sdk/sdk-by-examples/simple-governance/app-init.md

```diff
@@ -1,4 +1,4 @@
-## Application initialization 
+# Application initialization 
```

Comment:
Capitalize both words. Looks nicer in sidebar

Reply:
y